libraries/core-react/src/components/Tooltip/Tooltip.tokens.ts

```diff
@@ -16,40 +19,37 @@ const {
   },
 } = tokens
 
-export type Tooltip = {
-  typography: Typography
-  background: string
-  tooltip: {
-    minHeight: string
-  }
-  arrow: {
-    width: string
-    height: string
-    placement: string
-  }
-  spacings: Spacing
-  borderRadius: string
-}
+type TooltipToken = ComponentToken
 
-export const tooltip: Tooltip = {
+export const tooltip: TooltipToken = {
   typography: {
     ...ui.tooltip,
-    color: '#fff',
+    color: white,
   },
   background,
-  tooltip: {
-    minHeight: spacingXlarge,
+  border: {
+    type: 'border',
+    radius: borderRadius,
   },
-  arrow: {
-    width: '6px',
-    height: spacingSmall,
-    placement: '-6px',
+  entities: {
+    tooltip: {
+      height: spacingXlarge,
+    },
+    arrow: {
+      width: '6px',
+      height: spacingSmall,
+      spacings: {
+        bottom: '-6px',
```

Comment:
Any reason you removed `spacingSmall` here?

Reply:
Which `spacingSmall` are you referring to? I haven't removed anything I think... 🤔

Reply:
```
spacings: {
    left: spacingSmall,
    right: spacingSmall,
    top: spacingSmall,
    bottom: spacingSmall,
 }
```
   But I see now these were not really removed, I was confused by the new arrow spacings 😆